resources/views/util/message.blade.php

```diff
@@ -0,0 +1,6 @@
+@if ($message = Session::get('success'))
+<div class="alert alert-success alert-block">
+    <button type="button" class="close" data-dismiss="alert">×</button>
+        <strong>{{ $message }}</strong>
+</div>
+@endif
```

Comment:
Maybe we can do a more general file. It's fine by now.

Reply:
So is it good?

Reply:
Yes